main/webapp/modules/core/scripts/index/open-project-ui.js

```diff
@@ -132,18 +172,40 @@ Refine.OpenProjectUI.prototype._addTagsListAnimation = function() {
     });
 };
 
-Refine.OpenProjectUI.prototype._fetchProjects = function() {
-    var self = this;
-    $.ajax({
-            type : 'GET',
-            url : "command/core/get-all-project-metadata",
-            dataType : 'json',
-            success : function(data) {
-                    self._renderProjects(data);
-                    self.resize();
-            },
-            data : {},
-            async : false
+Refine.OpenProjectUI.prototype._searchAnimation = function() {
+    var search = $('#searchIcon');
+    var form = $('.header-search-box');
+    var icon = $('.magnifying_glass');
+    search.click(function () {
```

Comment:
after clicking search, I would also like to see the focus go to the input form and have cursor already there to begin typing.